templates/mod/php5.conf.erb

```diff
@@ -14,8 +14,9 @@
 #
 # Cause the PHP interpreter to handle files with a .php extension.
 #
-AddHandler php5-script <%= @extensions.flatten.compact.join(' ') %>
-AddType text/html .php
+<FilesMatch ".+\.(<%= @extensions.flatten.compact.join('|').gsub('.','') %>)$">
```

Comment:
I've modified this to use `Regexp.escape` to allow even weird extensions (e.g. `.en.php`) to be passed through unchanged. The new PR is #1277